main/webapp/modules/core/scripts/views/data-table/menu-reconcile.js

```diff
@@ -55,6 +55,33 @@ DataTableColumnHeaderUI.extendMenu(function(column, columnHeaderUI, menu) {
     );
   };
 
+  var doAddColumnWithUrlOfMatchedEntities = function AddColumnWithUrlOfMatchedEntities (columnName,dismissDialog) {
```

Comment:
Here you can simplify the way you declare this function:
```suggestion
  var doAddColumnWithUrlOfMatchedEntities = function (columnName,dismissDialog) {
```